skills/lark-doc/references/lark-doc-xml.md

```
@@ -52,3 +52,30 @@
 - 文本转义：`<` → `&lt;`，`>` → `&gt;`，`&` → `&amp;`，换行符 `\n` → `<br/>`。
 - 错误：`&lt;p&gt;内容&lt;/p&gt;`
 - 正确：`<p>A &amp; B 的对比：1 &lt; 2</p>`
+
+
+## 转义规则
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)